xml/System.Windows.Controls.Ribbon/StringCollectionConverter.xml

```diff
@@ -1,173 +1,175 @@
+      <MemberSignature Language="DocId" Value="M:System.Windows.Controls.Ribbon.StringCollectionConverter.ConvertTo(System.ComponentModel.ITypeDescriptorContext,System.Globalization.CultureInfo,System.Object,System.Type)" />
+      <MemberSignature Language="VB.NET" Value="Public Overrides Function ConvertTo (context As ITypeDescriptorContext, culture As CultureInfo, value As Object, destinationType As Type) As Object" />
+      <MemberSignature Language="F#" Value="override this.ConvertTo : System.ComponentModel.ITypeDescriptorContext * System.Globalization.CultureInfo * obj * Type -&gt; obj" Usage="stringCollectionConverter.ConvertTo (context, culture, value, destinationType)" />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; override System::Object ^ ConvertTo(System::ComponentModel::ITypeDescriptorContext ^ context, System::Globalization::CultureInfo ^ culture, System::Object ^ value, Type ^ destinationType);" />
+      <MemberType>Method</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.Windows.Controls.Ribbon</AssemblyName>
+        <AssemblyVersion>4.0.0.0</AssemblyVersion>
+        <AssemblyVersion>5.0.0.0</AssemblyVersion>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Object</ReturnType>
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="context" Type="System.ComponentModel.ITypeDescriptorContext" />
+        <Parameter Name="culture" Type="System.Globalization.CultureInfo" />
+        <Parameter Name="value" Type="System.Object" />
+        <Parameter Name="destinationType" Type="System.Type" />
+      </Parameters>
+      <Docs>
+        <param name="context">An object that provides a format context.</param>
+        <param name="culture">The object to use as the current culture.</param>
+        <param name="value">The <see cref="T:System.Collections.Specialized.StringCollection" /> to convert.</param>
+        <param name="destinationType">The type to convert the <paramref name="value" /> to.</param>
+        <summary>Converts an object of type <see cref="T:System.Collections.Specialized.StringCollection" /> to a <see cref="T:System.String" /> or <see cref="T:System.ComponentModel.Design.Serialization.InstanceDescriptor" />.</summary>
+        <returns>The converted value.</returns>
+        <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException">
+          <paramref name="value" /> is <see langword="null" />.  
+  
+ -or-  
+  
+ <paramref name="destinationType" /> is <see langword="null" />.</exception>
+        <exception cref="T:System.NotSupportedException">A NotSupportedException is thrown if the example object is null
+            or if the destinationType isn't one of the valid destination types.</exception>

```

Comment:
This exception contradicts the already existing `ArgumentNullException` above.

Reply:
Maybe this is a case of a breaking change between .NET Framework and .NET Core? In which case, the difference should be explicitly mentioned.

Reply:
This looks correct.  ArgumentNullException when Value parameter is null and NotSupportedException when destinationType is null.